src/services/SyncService.js

```diff
@@ -165,7 +162,7 @@ class SyncService {
 					this.#sendIntervalId = null
 					this._sendSteps(getSendable).then(resolve).catch(reject)
 				}
-			}, 200)
+			}, 50)
```

Comment:
This could have an impact on the resource demand due to increased push requests on the server. I think the PR itself should also be fine with the existing 200ms. Any reason to decrease that specific to the cursor hiding?

Reply:
This is a bit tricky as we change the initial waiting. Before if there was no request pending we would just send it immediately - so the 200 ms had no effect at all. Now we also delay this initial message.

I remember that 200ms are roughly what the human brain starts to notice as a delay. But that is only if one is waiting for a direct response from the UI. This will only delay the sync between different parties. So 200 ms are probably okay there as well.

I'd say we go with the 200 ms and if we notice usability issues with it we can change it to 50 ms on the initial request and 200 when another request is pending.

Reply:
Okay... so i gave this a try locally with 200 and noticed that the polling delay still is the main delay one notices. If I type fairly fast 200ms will result in two keystrokes getting combined in one push request. That seems totally reasonable. If this actually reduces the number of requests send - even better.